sc.py

```diff
@@ -0,0 +1,42 @@
+from microcosm.opaque import Opaque
```

Comment:
Ooops. I didn't mean to check this in. This is my extended example from the comment.